python/lsst/afw/multiband.py

```diff
@@ -83,10 +83,10 @@ def clone(self, deep=True):
         pass
 
     @property
-    def filters(self):
-        """List of filter names for the single band objects
+    def bands(self):
+        """List of band names for the single band objects
         """
-        return self._filters
```

Comment:
I'd suggest keeping this for now, but deprecating it while adding `bands` as a new property.

Reply:
This appeared a few rows below I intended. I meant to keep `filters` as a property, not `_filters` attribute.